mlrun/kfpops.py

```diff
@@ -41,8 +41,8 @@
 
 # default KFP artifacts and output (ui metadata, metrics etc.)
 # directories to /tmp to allow running with security context
-KFPMETA_DIR = os.environ.get("KFPMETA_OUT_DIR", "/tmp")
-KFP_ARTIFACTS_DIR = os.environ.get("KFP_ARTIFACTS_DIR", "/tmp")
+KFPMETA_DIR = "/tmp"
+KFP_ARTIFACTS_DIR = "/tmp"
```

Comment:
I'm ok with removing this, but I think we need at least a RN that notes we no longer allow the use of these env variables, just in case someone was doing it.